src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1586,6 +1586,7 @@ export class SavedObjectsRepository {
         [type]: attributes,
         updated_at: time,
         ...(Array.isArray(references) && { references }),
+        ...(workspaces && { workspaces }),
```

Comment:
We need to consider if we need to do conflict check in [conflict wrapper](https://github.com/ruanyl/OpenSearch-Dashboards/blob/workspace/src/plugins/workspace/server/saved_objects/saved_objects_wrapper_for_check_workspace_conflict.ts) and permission check in [permission wrapper](https://github.com/ruanyl/OpenSearch-Dashboards/blob/workspace/src/plugins/workspace/server/saved_objects/workspace_saved_objects_client_wrapper.ts) if we support bulkUpdate with workspaces field.

Reply:
I will use addToWorkspace function instead of bulkUpdate so no conflict and permission validation